### ISSUE TITLE
BUG: Notices on IDX Import Tool

### ISSUE THREAD

**Issue by bhubbard** [agentevolution/wp-listings#65]
```php
[28-Mar-2016 17:18:53 UTC] PHP Warning:  in_array() expects parameter 2 to be array, null given in wp-content/plugins/wp-listings/includes/class-listing-import.php on line 78
[28-Mar-2016 17:18:53 UTC] PHP Warning:  in_array() expects parameter 2 to be array, null given in wp-content/plugins/wp-listings/includes/class-listing-import.php on line 90
[28-Mar-2016 17:18:53 UTC] PHP Warning:  in_array() expects parameter 2 to be array, null given in wp-content/plugins/wp-listings/includes/class-listing-import.php on line 137
[28-Mar-2016 17:18:53 UTC] PHP Warning:  in_array() expects parameter 2 to be array, null given in wp-content/plugins/wp-listings/includes/class-listing-import.php on line 142
[28-Mar-2016 17:18:53 UTC] PHP Notice:  Undefined index: status in wp-content/plugins/wp-listings/includes/class-listing-import.php on line 142
[28-Mar-2016 17:18:53 UTC] PHP Warning:  in_array() expects parameter 2 to be array, null given in wp-content/plugins/wp-listings/includes/class-listing-import.php on line 78
[28-Mar-2016 17:18:53 UTC] PHP Warning:  in_array() expects parameter 2 to be array, null given in wp-content/plugins/wp-listings/includes/class-listing-import.php on line 90
[28-Mar-2016 17:18:53 UTC] PHP Warning:  in_array() expects parameter 2 to be array, null given in wp-content/plugins/wp-listings/includes/class-listing-import.php on line 137
[28-Mar-2016 17:18:53 UTC] PHP Warning:  in_array() expects parameter 2 to be array, null given in wp-content/plugins/wp-listings/includes/class-listing-import.php on line 142
[28-Mar-2016 17:18:53 UTC] PHP Notice:  Undefined index: status in wp-content/plugins/wp-listings/includes/class-listing-import.php on line 142
[28-Mar-2016 17:18:53 UTC] PHP Warning:  in_array() expects parameter 2 to be array, null given in wp-content/plugins/wp-listings/includes/class-listing-import.php on line 78
[28-Mar-2016 17:18:53 UTC] PHP Warning:  in_array() expects parameter 2 to be array, null given in wp-content/plugins/wp-listings/includes/class-listing-import.php on line 90
[28-Mar-2016 17:18:53 UTC] PHP Warning:  in_array() expects parameter 2 to be array, null given in wp-content/plugins/wp-listings/includes/class-listing-import.php on line 137
[28-Mar-2016 17:18:53 UTC] PHP Warning:  in_array() expects parameter 2 to be array, null given in wp-content/plugins/wp-listings/includes/class-listing-import.php on line 142
```

**Comment by bhubbard** [comment]
```php
[28-Mar-2016 17:25:32 UTC] PHP Notice:  Undefined index: status in wp-content/plugins/wp-listings/includes/class-listing-import.php on line 142
[28-Mar-2016 17:25:32 UTC] PHP Notice:  Undefined index: status in wp-content/plugins/wp-listings/includes/class-listing-import.php on line 142
[28-Mar-2016 17:25:32 UTC] PHP Notice:  Undefined index: status in wp-content/plugins/wp-listings/includes/class-listing-import.php on line 142
[28-Mar-2016 17:25:32 UTC] PHP Notice:  Undefined index: status in wp-content/plugins/wp-listings/includes/class-listing-import.php on line 142
[28-Mar-2016 17:25:32 UTC] PHP Notice:  Undefined index: status in wp-content/plugins/wp-listings/includes/class-listing-import.php on line 142
[28-Mar-2016 17:25:32 UTC] PHP Notice:  Undefined index: status in wp-content/plugins/wp-listings/includes/class-listing-import.php on line 142
```

**Comment by davebonds** [comment]
Fixed in f2dadc8d2a8aea9948eba0a5d39ea29681201e04
